apps/ui/src/components/SpacesListItem.vue

```diff
@@ -1,16 +1,35 @@
 <script lang="ts" setup>
 import { _n } from '@/helpers/utils';
 import { offchainNetworks } from '@/networks';
-import { Space } from '@/types';
+import { NetworkID } from '@/types';
 
-const props = defineProps<{ space: Space }>();
+const props = withDefaults(
+  defineProps<{
+    space: {
+      id: string;
+      name: string;
+      about?: string;
+      avatar: string;
+      cover: string;
+      proposal_count: number;
+      vote_count: number;
+      turbo: boolean;
+      verified: boolean;
+      network: NetworkID;
+      snapshot_chain_id?: number;
+    };
```

Comment:
I think better to use 

```suggestion
    space: Space | RelatedSpace;
```

and in `RelatedSpace` we can add `about?: string;`